src/backend/libinput/mod.rs

```diff
@@ -614,9 +614,7 @@ impl<S: Session> libinput::LibinputInterface for LibinputSessionInterface<S> {
         Ok(unsafe { OwnedFd::from_raw_fd(fd) })
     }
 
-    fn close_restricted(&mut self, fd: OwnedFd) {
-        let _ = self.0.close(fd.into_raw_fd());
-    }
+    fn close_restricted(&mut self, _fd: OwnedFd) {}
```

Comment:
This is unfortunately not correct, session-close does more then just call `close` on the fd. We need to convert back here.

Reply:
Ah right. I misread that as just a call to nix's `close`.